packages/features/vault/src/side-drawers/components/vault-side-drawer.tsx

```diff
@@ -48,29 +47,46 @@ export const VaultSideDrawer = ({
       updatedVaultAccount,
     },
   } = useContext(VaultActionContext);
+  const useVaultData = useVault(vaultAddress);
   const currentVaultAddress = vaultAddress || '';
   const canSubmit = buildTransactionCall ? true : false;
 
+  const messageValues = {
+    [VAULT_ACTIONS.CREATE_VAULT_POSITION]: useVaultData?.minDepositRequired,
+  };
+
   const formattedMaturity = vaultAccount?.maturity
     ? formatMaturity(vaultAccount?.maturity)
     : '';
 
+  const handleCancel = () => {
+    clearSideDrawer(`/vaults/${vaultAddress}`);
+    if (vaultAction === VAULT_ACTIONS.WITHDRAW_VAULT) {
+      updateState({
+        maxWithdraw: false,
+        withdrawAmount: undefined,
+        withdrawAmountString: '',
```

Comment:
Don't set withdrawAmountString here, it should be unused

Reply:
When the user cancels, I think the vaultAction should be set to undefined every time? Does that happen elsewhere?